app/assets/javascripts/alchemy/alchemy.link_dialog.js.coffee

```diff
@@ -94,7 +94,7 @@ class window.Alchemy.LinkDialog extends Alchemy.Dialog
         urlname = $element.val()
         $.get Alchemy.routes.api_pages_path,
           q:
-            urlname_eq: urlname.replace(/^\/([a-z]{2}(-[A-Z]{2})?\/)?/, '')
+            urlname_eq: urlname.replace(/^\/?([a-z]{2}(-[A-Z]{2})?\/)?(.+?)\/?$/, '$3')
```

Comment:
Hm so now the leading slash is optional?

Reply:
Reverted that